CONTRIBUTING.md

```diff
@@ -61,6 +61,19 @@ tabs). Format your code assuming it will be read in a window 100
 columns wide. Wrap code at 100 characters and comments at 80 unless doing so
 makes the code less legible.
 
+When wrapping function signatures that do not fit on one line,
+put the name, arguments and return types on separate lines, with the closing `)`
+at the same indentation as `func`. Example:
+```go
+func (s *someType) myFunctionName(
+    arg1 somepackage.SomeArgType, arg2 int, arg3 somepackage.SomeOtherType,
+) (somepackage.SomeReturnType, error) {
+    ...
+}
+```
+If the arguments list exceeds 100 characters, switch to one argument per line. 
```

Comment:
Dunno - have to see what it looks like rendered.

On Fri, Mar 25, 2016 at 12:12 PM, David Taylor <notifications@github.com>
wrote:

> In CONTRIBUTING.md
> <https://github.com/cockroachdb/cockroach/pull/5578#discussion_r57458816>:
>
> > @@ -61,6 +61,19 @@ tabs). Format your code assuming it will be read in a window 100
> >  columns wide. Wrap code at 100 characters and comments at 80 unless doing so
> >  makes the code less legible.
> >
> > +When wrapping function signatures that do not fit on one line,
> > +put the name, arguments and return types on separate lines, with the closing `)`
> > +at the same indentation as `func`. Example:
> > +```go
> > +func (s *someType) myFunctionName(
> > +    arg1 somepackage.SomeArgType, arg2 int, arg3 somepackage.SomeOtherType,
> > +) (somepackage.SomeReturnType, error) {
> > +    ...
> > +}
> > +```
> > +If the arguments list exceeds 100 characters, switch to one argument per line.
>
> Hm, should I just link to gist/playground links for some of those? Or
> inline all of them? seems like it it might be a bit verbose to do all
> inline.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/5578/files/f30c68b5b082a459aa0cef893d916f6420b221c6#r57458816>
>